CHANGELOG.md

```diff
@@ -10,11 +10,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Support for item.Attachment:Mail restore
+- Errors from duplicate names in Exchange Calendars
+
+### Breaking Change: Exchange Calendar Storage
+Exchange Calendars storage entries now construct their paths using container IDs instead of display names.  This fixes cases where duplicate display names caused system failures.  The migration conflicts with existing calendar backups, producing the following problems:
+
+1. On the following backups, all Calendars and Calendar Events will produce duplicate storage entries.
+2. Attempting to restore multiple duplicate storage entries (ex, from an untargeted restore command like: `corso restore exchange --backup id`) will cause errors due to duplicate item collisions.
+
+To avoid these issues, we recommend running a backup with the flag --disable-incrementals.  Ex:
+```
+corso backup exchange --user '*' --data events --disable-incrementals
```

Comment:
Do we have duplicate entries from previous backups without this flag? It seems like incrementals are enough to avoid that to begin with no?

Reply:
Went and did some manual testing, and right you are!  Looks like our concerns about duplicates were misinformed.  Which mean, I guess, that these aren't breaking changes after all...